packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1079,11 +1079,30 @@ export class Profiles extends ProfilesCache {
                 cmdArgs[prop] = serviceProfile.profile[prop] ? serviceProfile.profile[prop] : baseProfile.profile[prop];
             }
             if (baseProfile) {
-                cmdArgs.tokenType = serviceProfile.profile.tokenType ? serviceProfile.profile.tokenType: baseProfile.profile.tokenType;
-                cmdArgs.tokenValue = serviceProfile.profile.tokenValue ? serviceProfile.profile.tokenValue: baseProfile.profile.tokenValue;
+                cmdArgs.tokenType = serviceProfile.profile.tokenType
+                    ? serviceProfile.profile.tokenType
+                    : baseProfile.profile.tokenType;
+                cmdArgs.tokenValue = serviceProfile.profile.tokenValue
+                    ? serviceProfile.profile.tokenValue
+                    : baseProfile.profile.tokenValue;
             }
             if (commonApi.getSessionFromCommandArgument) {
-                session = await commonApi.getSessionFromCommandArgument(cmdArgs);
+                if (cmdArgs.tokenType === undefined || cmdArgs.tokenValue === undefined) {
+                    this.log.debug(
+                        localize("getCombinedProfile.noToken1", "Profile ") +
+                            baseProfile.name +
+                            " " +
+                            serviceProfile.name +
+                            localize(
+                                "getCombinedProfile.noToken2",
+                                " is not authorized. Please check the connection and try again"
+                            )
+                    );
```

Comment:
Just a note, you should be able to write the localization as:
```
this.log.debug(
    localize(
        "getCombinedProfile.noToken",
        "Profile {0} {1} is not authorized. Please check the connection and try again.",
         baseProfile.name,
         serviceProfile.name
    )
);
```

Keeping the message in a single localization string will make it easier for translators to understand what we are trying to say (in addition to looking neater).
- (Remember to re-run `yarn run vscode:prepublish` to update the localization strings if you decide to use this suggestion1)

Reply:
Thanks for the review @lauren-li. I pushed new changes to apply what you said